packages/cdk/lib/constructs/batch.ts

```diff
@@ -224,4 +220,30 @@ export class Batch extends Construct {
       },
     });
   }
+
+  private getLaunchTemplateProps(launchTemplateData?: string, resourceTags?: { [p: string]: string }): CfnLaunchTemplateProps | undefined {
```

Comment:
for naming consistency with other methods could you call this `renderLaunchTemplateProps`

Reply:
@markjschreiber done